docs/index.rst

```diff
@@ -23,6 +23,18 @@ Sylius Documentation
 
    For tips on using only some bundles of Sylius head to Bundles and Components docs.
 
+Getting Started with Sylius
+---------------------------
+
+Getting Started with Sylius
```

Comment:
Well... I think we need sth more here :D

Reply:
xD